shell/platform/tizen/channels/platform_channel.cc

```diff
@@ -156,8 +158,10 @@ void PlatformChannel::HandleMethodCall(
 }
 
 void PlatformChannel::SystemNavigatorPop() {
-  if (window_) {
+  if (view_->GetType() == TizenViewType::kWindow) {
```

Comment:
`view_` could be null if the app is running in headless mode.

Reply:
`PlatformChannel` is not created in headless mode. could you check this again?

Reply:
I told because the documentation in `platform_channel.h` said so. Please remove the documentation if `view_` is guaranteed to be non-null.

You might also update the documentation in `window_channel.h` since `window_` is not an instance of type `TizenRenderer`.